Mlem/Views/Tabs/Posts/Components/Post/Components/Instance and Community List/Accounts Page.swift

```diff
@@ -46,6 +52,13 @@ struct AccountsPage: View
                             }
                         }
                         .onDelete(perform: deleteAccount)
+                        .navigationDestination(isPresented: accountNavigationBinding(), destination: {
+                            CommunityView(account: accountsTracker.savedAccounts.first!, community: nil)
```

Comment:
No Force unwrapping please.

Reply:
I just changed it. Please let me know if I can do it better 👍